subprojects/model-core/src/main/java/org/gradle/util/internal/ConfigureUtil.java

```diff
@@ -155,6 +158,30 @@ private static <T> void configureTarget(Closure configureClosure, T target, Conf
         new ClosureBackedAction<T>(withNewOwner, Closure.OWNER_ONLY, false).execute(target);
     }
 
+    public static Object unwrapBean(Object bean) {
```

Comment:
Is `ConfigureUtil` right place for this method? Asking since other methods configure something and this one unwraps a bean.

Reply:
Good idea, let's move it to `ScriptSourceAwareImplementationIdentifier`.

Reply:
Handled in https://github.com/gradle/gradle/pull/22759.